### PR TITLE
AutoEdits: Add 2 tools, tweak some others

### PR DESCRIPTION
**Pull request by DannyS712**
Commit 1: Update page curation with namespace 4 and pagetriage tag
Commit 2: Add userRightsManager for enwiki, in namespaces 3 and 4
Commit 3: Add Replacer tool for eswiki
Commit 4: Expand wikidata merge tool to include Special:MergeItems
Commit 5: Combine different commons global replace tools into 1 global rule
Commit 6: Rename Sagittarius to Capricorn, update regex and link

Bug: T237217
Bug: T237174
Bug: T237076
Bug: T231709
Bug: T236719
Bug: T237458

### CODE REVIEW

File: config/semi_automated.yml [hunk 1]
```diff
@@ -520,9 +523,9 @@ parameters:
             regex: 'MediaWiki_talk:Gadget-labelLister.js\|labelLister'
             link: MediaWiki talk:Gadget-labelLister.js
             namespaces: [ 0, 120 ]
-        Merge.js:
-            regex: '\[\[MediaWiki:Gadget-Merge\.js\|merge\.js\]\]'
-            link: MediaWiki talk:Gadget-Merge.js
+        Merge:
+            regex: '\[\[MediaWiki:Gadget-Merge\.js\|merge\.js\]\]|^Merged Item'
```
Comment (MusikAnimal) [inline config/semi_automated.yml]: I had to query the database to find out, but looks like the summaries for automatic merges are actually like:
```
/* wbmergeitems-to:0||Q5610520 */
```
or
```
/* wbmergeitems-from:0||Q25416796 */
```
This I'm sure is for localization purposes.

Something like `\/* wbmergeitems` should work.

**Review by MusikAnimal — CHANGES_REQUESTED** [review]
Just one issue with Merge, everything else looks great!

The Sagittarius/Capricorn change got me thinking... I wonder if we should include an `aliases` option, so that older incoming links still work. Not important right now.

File: config/semi_automated.yml (hunk 1 above)
Comment (DannyS712) [reply config/semi_automated.yml]: Done (except I used `\/\* wbmergeitems`)

**Review by MusikAnimal — APPROVED** [review]
Thank you as always!

**Pull request merged**